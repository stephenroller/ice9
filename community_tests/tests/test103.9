# var a : int \n a := 0; do a < 5 -> a := a + 1; od		succeed

var a : int

a := 0; 

do a < 5 -> 
	a := a + 1;
	write a;
od
