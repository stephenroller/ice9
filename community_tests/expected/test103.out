Loading file: generated/test103.tm
1 
2 
3 
4 
5 
Number of instructions executed = 203
